packages/framework/src/Component/Domain/Domain.php

```diff
@@ -215,4 +218,29 @@ public function isB2b(): bool
     {
         return $this->getCurrentDomainConfig()->isB2b();
     }
+
+    /**
+     * @return int[]
+     */
+    public function getAdminEnabledDomainIds(): array
+    {
+        $selectedDomainIds = $this->administratorFacade->getCurrentlyLoggedAdministrator()->getDisplayOnlyDomainIds();
+
+        return count($selectedDomainIds) > 0 ? $selectedDomainIds : $this->getAllIds();
+    }
+
+    /**
+     * @return string[]
+     */
+    public function getAdminEnabledLocales(): array
+    {
+        $locales = [];
+
+        foreach ($this->getAllIds() as $selectedDomainId) {
```

Comment:
should not here be `getAdminEnabledDomainIds`?

Reply:
the method is unused, I have removed it